applications/webhooks/src/webhooks_channel_util.erl

```diff
@@ -56,7 +56,12 @@ format_event(JObj, AccountId, <<"CHANNEL_DESTROY">>) ->
                       ,{<<"ringing_seconds">>, kz_call_event:ringing_seconds(JObj)}
                       ,{<<"billing_seconds">>, kz_call_event:billing_seconds(JObj)}
                      ]
-                   ).
+                   );
+format_event(JObj, AccountId, <<"CHANNEL_PARK">>) ->
```

Comment:
@macpie 
Is this necessary?  I assume this was from an earlier attempt to use CHANNEL_PARK.  If it is you should remove the io:format ;)